src/renderer/ts/Game.ts

```diff
@@ -230,17 +230,15 @@ class Game implements Eventful, Drawable {
    * @param {GameState} state The game state to active
    */
   private lock(state: GameState) {
-    if (this.state === GameState.Play) {
-      this.state = state;
-      this.player.lock();
-
-      if (state !== GameState.Inventory) {
-        this.inventory.lock();
-      } 
-      if (state !== GameState.StartMenu) {
-        this.menu.lock();
-      }
+    if (this.state !== GameState.Play) {
+      return;
     }
+
+    this.state = state;
+    this.player.lock();
+
+    state !== GameState.Inventory ? this.inventory.lock() : null;
```

Comment:
@ChristianDalfonso please change these two lines back. While it's less code, it becomes harder to understand the intent of what its doing.